src/dmd/lexer.d

```diff
@@ -247,10 +262,20 @@ class Lexer
                 p++;
                 if (*p != '\n') // if CR stands by itself
                     endOfLine();
+                if (tokenizeNewlines)
+                {
+                    t.value = TOK.endOfLine;
```

Comment:
and now turn off tokenizeNewLines here.

Reply:
I've just added `scope(exit) tokenizeNewlines = false` in the case scope.